fix(input): textarea right space

juliodialpad opened the pull request:
## Description

- Added absolute positioning to right icon on textarea
- Updated documentation examples to `dt-input` component

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)
![](https://media.giphy.com/media/sdqpqwEbzsZQUdf4Hp/giphy.gif)

iropolo commented:
hmm, doubt on this but seems like is intended behaviour

<img width="237" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/f6be5269-5cd3-4a31-925b-fe54322bf6e2">

iropolo commented:
> > hmm, doubt on this but seems like is intended behaviour
> > image
> > Would be cool to have a designer review here.
> 
> Yeah we need to adjust right padding in this case.

ye but... that is the actually design...

<img width="525" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/4bd0f69e-13e2-4622-834b-1476e402606c">

I mean... this PR is for a reason ..

braddialpad commented:
> ye but... that is the actually design...
> I mean... this PR is for a reason ..

padding will not move the position of the resizer, only changing the width will.

juliodialpad commented:
- Added padding to the right and increased the right position of the icon to avoid overlapping the scrollbar.
- Loaded dialtone directly from `lib/dist/css/dialtone.css/` again as vuepress wasn't watching the changes, there'd be no problem with the autoprefixer since I added ours to gulp.

braddialpad reviewed (approved):
I think this should work reasonably enough for now

ninarepetto commented:
How is this plugin used in agent-assist-widget?

braddialpad commented:
> How is this plugin used in agent-assist-widget?

simply added to the plugins array in postcss.config.js

All this plugin does is change the tokens files to be `:host` instead of `:root` so the tokens are applied to the shadow dom host element.
